### ISSUE TITLE
Rotating single-sig delegate identifier

### ISSUE THREAD

**Issue by psteniusubi** [WebOfTrust/signify-ts#159]
The integration test to reproduce this issue is in PR #160 and here

https://github.com/psteniusubi/signify-ts/blob/feat-integration-test-singlesig-drt/examples/integration-scripts/singlesig-drt.test.ts

The `identifiers.rotate` command fails with 500 server error

Keria shows a stack trace

```
signify-ts-keria-1         | 2023-12-02 09:12:11 [FALCON] [ERROR] PUT /identifiers/delegate1 => Traceback (most recent call last):
signify-ts-keria-1         |   File "/keria/venv/lib/python3.10/site-packages/keri/app/habbing.py", line 2443, in processEvent
signify-ts-keria-1         |     self.kvy.processEvent(serder=serder, sigers=sigers)
signify-ts-keria-1         |   File "/keria/venv/lib/python3.10/site-packages/keri/core/eventing.py", line 2994, in processEvent
signify-ts-keria-1         |     kever.update(serder=serder, sigers=sigers, wigers=wigers,
signify-ts-keria-1         |   File "/keria/venv/lib/python3.10/site-packages/keri/core/eventing.py", line 1940, in update
signify-ts-keria-1         |     raise ValidationError("Attempted non delegated rotation on "
signify-ts-keria-1         | keri.kering.ValidationError: Attempted non delegated rotation on delegated pre = EA8nuk8Z3ZwhL4PmECqNs6w4hUk47RGeiRTn0UKvRf74 with evt = {'v': 'KERI10JSON000160_', 't': 'rot', 'd': 'EA0dFkf7S1w3uJANMeZlcWAzTQRbgqURdjpFxSbjgekk', 'i': 'EA8nuk8Z3ZwhL4PmECqNs6w4hUk47RGeiRTn0UKvRf74', 's': '1', 'p': 'EA8nuk8Z3ZwhL4PmECqNs6w4hUk47RGeiRTn0UKvRf74', 'kt': '1', 'k': ['DLmuY28mF6Golgq2SJVENE4iLXqNc2nMcb7bh6eiLSqZ'], 'nt': '1', 'n': ['ELY_6-spjz3o-ceMXis4i81maZaRgzk45vjVWKMaxOAs'], 'bt': '0', 'br': [], 'ba': [], 'a': []}.
signify-ts-keria-1         |
signify-ts-keria-1         | During handling of the above exception, another exception occurred:
signify-ts-keria-1         |
signify-ts-keria-1         | Traceback (most recent call last):
signify-ts-keria-1         |   File "falcon/app.py", line 365, in falcon.app.App.__call__
signify-ts-keria-1         |   File "/keria/src/keria/app/aiding.py", line 470, in on_put
signify-ts-keria-1         |     op = self.rotate(agent, name, body)
signify-ts-keria-1         |   File "/keria/src/keria/app/aiding.py", line 506, in rotate
signify-ts-keria-1         |     hab.rotate(serder=serder, sigers=sigers)
signify-ts-keria-1         |   File "/keria/venv/lib/python3.10/site-packages/keri/app/habbing.py", line 2404, in rotate
signify-ts-keria-1         |     self.processEvent(serder, sigers)
signify-ts-keria-1         |   File "/keria/venv/lib/python3.10/site-packages/keri/app/habbing.py", line 2445, in processEvent
signify-ts-keria-1         |     raise kering.ConfigurationError(f"Improper Habitat event type={serder.ked['t']} for "
signify-ts-keria-1         | keri.kering.ConfigurationError: Improper Habitat event type=rot for pre=EA8nuk8Z3ZwhL4PmECqNs6w4hUk47RGeiRTn0UKvRf74.
signify-ts-keria-1         |
```

**Comment by m00sey** [comment]
This should be a `drt` not a `rot`

**Comment by pfeairheller** [comment]
Fixed with PR #197
